java/compaction/compaction-job-execution/src/test/java/sleeper/compaction/job/execution/CompactSortedFilesEmptyOutputIT.java

```diff
@@ -76,8 +80,10 @@ void shouldMergeFilesCorrectlyWhenAllAreEmpty() throws Exception {
         assignJobIdToInputFiles(stateStore, compactionJob);
 
         // When
-        CompactSortedFiles compactSortedFiles = createCompactSortedFiles(schema);
-        RecordsProcessed summary = compactSortedFiles.compact(compactionJob);
+        DefaultSelector compactSortedFiles = createCompactionSelector(schema,
+            HadoopConfigurationProvider.getConfigurationForECS(instanceProperties));
+        sleeper.compaction.job.CompactionRunner runner = compactSortedFiles.chooseCompactor(compactionJob);
```

Comment:
The full path to the CompactionRunner class looks a bit odd here, it looks like it can be removed